.gitignore

```diff
@@ -0,0 +1,7 @@
+src/org/zaproxy/zap/extension/bruteforce/resources/help_*
+build/build/
+build/log
+build/log1
```

Comment:
Is this entry and the above really needed? Do you know what program is creating them?